seguro/common/store.py

```diff
@@ -54,6 +62,16 @@ def __init__(
             region=region,
         )
 
+        self.logger = seguro.common.logger.init_logger(
```

Comment:
Maybe its sufficient to define a logger variable within the scope of the file (not a member of the class)?
Otherwise each client instance opens the same file?
Or at least a handler common to all client instances?

Reply:
I reworked the instantiation of loggers a bit to structure the logs more clearly:

- Broker and Store Clients are initialized with a `uid` (either manually or a automatically created one based on MAC and current time) with is used in the name of the log-files

As a result, client instances with (persistent) manually defined `uids` can log into the same file between different runs. Whereas client instances created without manually defined uids will not overwrite log files of other instances.

- For `file_loggers`, all logs will be written to `/log/<client_type>/<uid>.log`
- For `store_loggers`, all logs will be written to `/<S3_BUCKET>/logs/<uid>.log`